### PR TITLE
Make sentinels a list, and add :metal: to default sentinels.

### PR DESCRIPTION
**Pull request by jamiefolsom**
Hey @justinabrahms -- thoughts? Thanks!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -33,15 +33,16 @@ def main(sentinel, repos, token, since):
...
+                if sentinel in comment.body:
+                    total_reviews.append(comment.user.name)
 
     counter = collections.Counter(total_reviews)
     print '\n'.join(['%s: %s' % (x, y) for x,y in counter.most_common()])
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Count reviews on github")
-    parser.add_argument('--sentinel', dest='sentinel', default=':+1:', help='String to look for to count something as "reviewed"')
+    parser.add_argument('--sentinels', dest='sentinels', nargs="*", default='[":+1:", ":metal:"]', help='One or more strings to look for to count something as "reviewed"')
```
Comment (justinabrahms) [inline main.py:46]: How does this work on the command line?

```
--sentinels=:metal: :+1:
```

?

Does it have to be the last argument? How does it interpret spacing?

Comment (jamiefolsom) [inline main.py:46]: I'm assuming the behavior is set by argparse's nargs flag, and that indicates that this is a space-delimited list. See: https://docs.python.org/3/library/argparse.html#nargs ...So the argument would look like `--sentinels :+1: :metal:`

Comment (justinabrahms) [inline main.py:46]: Looks rad. Mind also updating the repo one as well and update the readme?

**Comment by jamiefolsom** [comment]
Sure -- is this what you mean: b5ba01c77e251a0c2efd24113c575b6622785de8?

**Comment by justinabrahms** [comment]
Yep. If you update the README, I'll merge. :D :metal:

**Comment by jamiefolsom** [comment]
Cool; did that. Thx.

**Comment by jamiefolsom** [comment]
:metal:

**Pull request closed without merging**